plugins/rapture/nexus-rapture-plugin/src/main/resources/static/rapture/NX/controller/dev/Logging.js

```diff
@@ -42,34 +38,64 @@ Ext.define('NX.controller.dev.Logging', {
   init: function () {
     var me = this;
 
+    // helper to lookup sink by name via Logging controller
+    function sink(name) {
+      return me.getController('Logging').getSink(name);
+    }
+
     me.listen({
       component: {
         'nx-dev-logging button[action=clear]': {
-          click: function(button) {
+          click: function (button) {
             me.getStore('LogEvent').removeAll();
           }
         },
-        'nx-dev-logging checkbox[itemId=console]': {
-          change: function(checkbox) {
-            NX.util.log.ConsoleSink.setEnabled(checkbox.getValue());
+
+        'nx-dev-logging combobox[itemId=threshold]': {
+          afterrender: function (combo) {
+            combo.select(me.getController('Logging').getThreshold());
+          },
+          select: function (combo) {
+            me.getController('Logging').setThreshold(combo.getValue());
           }
         },
-        'nx-dev-logging checkbox[itemId=remote]': {
-          change: function(checkbox) {
-            NX.util.log.RemoteSink.setEnabled(checkbox.getValue());
+
+        'nx-dev-logging checkbox[itemId=buffer]': {
+          afterrender: function (checkbox) {
+            checkbox.setValue(sink('store').enabled);
+          },
+          change: function (checkbox) {
+            sink('store').setEnabled(checkbox.getValue());
           }
         },
-        'nx-dev-logging combobox[itemId=threshold]': {
-          select: function(combo) {
-            me.getController('Logging').setThreshold(combo.getValue());
+
+        'nx-dev-logging numberfield[itemId=bufferSize]': {
+          afterrender: function (field) {
+            field.setValue(sink('store').maxSize);
           },
-          afterrender: function(combo) {
-            combo.select(me.getController('Logging').getThreshold());
+          change: function (field) {
```

Comment:
should be able to defer reacting to changes here if you demand an ENTER to set the value.
Switch to http://docs.sencha.com/extjs/4.2.3/#!/api/Ext.dom.Element-event-keypress and wrap the mutation of the sink with  if(event.getKey() == event.ENTER)